website/docs/advanced-concepts/multiple-environments.md

```diff
@@ -0,0 +1,95 @@
+
+# Multiple Environments
+
+Environments are container entities within a workspace that let you create different updates to different versions of your content types while still keeping them all in isolation. This process is often utilized when developing, identifying errors, or introducing new features without any concerns about how it might influence the current production.
+
+:::info
+Multiple Environments are only available on [Business Edition](https://www.appsmith.com/pricing). 
+:::
+
+Software developers can run numerous tests in these environments to filter out bugs and raise the application's overall quality. Different testing settings can be used to examine different aspects of a software application, including its performance, user interface, security, and more.
+
+## Environments on Appsmith
+There are two different environments that are available on Appsmith:
+
+### Staging environment
+Before launching an application, staging environments are created to evaluate builds, codes, and updates in an environment which is similar to a production environment. This is the area that developers take advantage of to inspect new aspects, debug, and explore their applications.
```

Comment:
let's stick to the context of datasources in Appsmith. and not discuss evaluating builds and codes. Let's also use the line shared in the first draft